### PR TITLE
Owner component workspaces by programs

### PR DESCRIPTION
**Pull request by paulfalgout**
Includes https://github.com/RoundingWell/care-ops-frontend/pull/1275

Shortcut Story ID: [sc-52510]

### CODE REVIEW

**Comment by paulfalgout** [comment]
This needs some more tests, but it might be good to go

**Review by zfixler — APPROVED** [review]

File: src/js/i18n/en-US.yml [hunk 1]
```diff
@@ -511,7 +511,6 @@ careOptsFrontend:
           system:
             actionCopiedFromProgram: '<strong>Action</strong> added from the { program } program'
             clinicianAssigned: '<strong>Owner</strong> changed to { to_name }'
-            created: '<strong>Action</strong> ({ team }) added'
```
Comment (shadowhand) [inline src/js/i18n/en-US.yml:514]: Why was this deleted?

Comment (paulfalgout) [reply src/js/i18n/en-US.yml:514]: At one point long ago "system" creates were filtered out by assuming that actions that came from program-actions were added by the system here: https://github.com/RoundingWell/care-ops-frontend/pull/1276/files#diff-fb5d041965ec94c9fc646e053667635bd8c984c188d4e687937fc816b8eacadbL249

Whenever source was added, the filter wasn't updated, and all events were duplicated are reformatted.  I don't think this one was ever intended to be displayed in the first place, and it is being filtered out.

related: https://github.com/RoundingWell/care-ops-frontend/pull/129

**Review by nmajor25 — APPROVED** [review]
